charts/stable/openemr/Chart.yaml

```
@@ -21,7 +21,7 @@ maintainers:
 dependencies:
   - name: common
     repository: https://library-charts.k8s-at-home.com
-    version: 2.5.0
+    version: 3.0.1
```

```suggestion
    version: 3.0.2
```
3.0.2 fixes a naming issue with dependency charts